test/null-and-undefined-test.js

```diff
@@ -69,28 +69,28 @@ buster.testCase('null & undefined keys & values', {
       done()
     },
 
-    'put() with null value works': function (done) {
+    'put() with null value causes error': function (done) {
```

Comment:
It's weird we still have these tests because this behavior is not defined by `levelup` but by the implementation of `abstract-leveldown` that is passed in.

Soon..